### ISSUE TITLE
why does batch normalization use feature wise normalization?

### ISSUE THREAD

**Issue by Tangzy7** [keras-team/keras#7709]
In paper of batch normalization, the normalization is sample-wise. Why is it feature-wise in keras?
